app/assets/stylesheets/dashboard.scss

```diff
@@ -32,11 +30,23 @@ $inactive-mixed: rgba(255, 165, 0, 0.4);
   }
 }
 
+@media only screen and (min-width: 40.063em) {
+
+  .content-wrapper {
+    margin-bottom: 9vw;
+    margin-top: 6vw;
+  }
+
+}
+
 //************************** NAVIGATION ***************************************
 
 nav {
-  font-family: 'Josefin Sans';
   text-transform: uppercase;
+
+  .name {
+  font-family: 'Josefin Sans';
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Prefer double-quoted strings